arch/Linux-x86-64-cuda.sopt

```diff
@@ -26,6 +26,7 @@ DFLAGS   = -D__CUDAPW -D__FFTW3
 CFLAGS   =  -O2
 CPPFLAGS = -traditional -C $(DFLAGS) -P -I/opt/intel/mkl/10.0.1.014/include/fftw
 CXXFLAGS = -O3 -I${CUDA_PATH}/include -std=c++11
+CXXOMPFLAGS = -fopenmp
 NVFLAGS	 = $(DFLAGS) -Xcompiler='-fopenmp' --std=c++11
```

Comment:
Can someone else comment on this one: in my understanding should `sopt` never use threads, yet here it seems that `.cu`'s were built with `-fopenmp`. Is that intentional?

And I must have missed it, but why do we need an extra flag `CXXOMPFLAGS` in DBCSR and can't use `CXXFLAGS`?

Reply:
> And I must have missed it, but why do we need an extra flag CXXOMPFLAGS in DBCSR and can't use CXXFLAGS?

Good question, now that you ask, I can't think of a good reason. `CXXFLAGS` appears in 2 rules in the Makefile: the [general rule for compiling cpp files](https://github.com/cp2k/dbcsr/blob/develop/Makefile#L497) and [the rule for compiling libcusmm.cpp](https://github.com/cp2k/dbcsr/blob/develop/Makefile#L500). Perhaps `CXXFLAGS` and `CXXOMPFLAGS` are separate so that `examples/dbcsr_example_3.cpp` doesn't get compiled with `-fopenmp` ?